src/game/client/gameclient.cpp

```diff
@@ -61,3 +62,33 @@ GameMessageDisposition CommandTranslator::Translate_Game_Message(const GameMessa
     return GameMessageDisposition();
 #endif
 }
+
+void GameClient::Add_Text_Bearing_Drawable(Drawable *drawable)
+{
+    m_drawableTB.push_back(drawable);
+}
+
+void GameClient::Remove_Drawable_From_Lookup_Table(Drawable *drawable)
+{
+    if (drawable != nullptr) {
+        m_drawableLUT[drawable->Get_ID()] = nullptr;
+    }
+}
+
+void GameClient::Add_Drawable_To_Lookup_Table(Drawable *drawable)
+{
+    if (drawable != nullptr) {
+        DrawableID id = drawable->Get_ID();
+
+        for (;;) {
+            if (id < m_drawableLUT.size()) {
+                break;
+            }
+
+            Drawable *d = nullptr;
+            m_drawableLUT.resize(2 * m_drawableLUT.size(), d);
```

Comment:
Can pass `nullptr` here directly for simplicity.

Reply:
fixed.